packages/create-package/create-package.js

```diff
@@ -55,7 +64,18 @@ const copyFile = (vars, sourcePath, destPath) => {
 	let content = fs.readFileSync(sourcePath, { encoding: "UTF-8" });
 	content = replaceVarsInFileContent(vars, content);
 	destPath = replaceVarsInFileName(vars, destPath);
+
 	fs.writeFileSync(destPath, content);
+
+	// Rename "gitignore" to ".gitignore" (npm init won't include ".gitignore", so we add it as "gitignore" and rename it later)
+	if (isGitIgnore(sourcePath)) {
```

Comment:
if {} else if {} no need to go through both checks

Reply:
will follow up on that, probably combining it something else